core/templates/dev/head/components/ck-editor-helpers/ck-editor-4-widgets.initializer.ts

```diff
@@ -61,6 +62,12 @@ angular.module('oppia').run([
             template: componentTemplate,
             draggable: false,
             edit: function() {
+              var contextIsExplorationPlayer = (
```

Comment:
Same thing, should be based on canWriteToFs. You may need to extract that into a common service that is accessible from both locations.